### DISCUSSION TITLE
youtube播放很慢是什么原因喃

### DISCUSSION THREAD

**Discussion by LaoHuan** [eycorsican/leaf#173] [Q&A]
用 aleaf，ss trojan 连上,youtube 的流只能最多1000Kpbs，其他都达到至少上万

**Comment by ghost** [comment]
> 用 aleaf，ss trojan 连上,youtube 的流只能最多1000Kpbs，其他都达到至少上万

有1000k不错了。我才100

**Comment by eycorsican** [comment]
服务端用 leaf 试试？
